conf/application.conf

```diff
@@ -75,7 +75,7 @@ metrics {
     host = graphite
     port = 2003
     prefix = play.${appName}.
-    enabled = true
+    enabled = false
```

Comment:
Why this is changed?

Reply:
This should be override in app-config-base .. done for local testing